charts/camunda-platform/templates/camunda/constraints.tpl

```diff
@@ -39,6 +39,16 @@ Fail with a message if the auth type is set to non-Keycloak and its requirements
   {{- end }}
 {{- end }}
 
+{{/*
+Fail with a message if Identity is disabled and identityKeycloak is enabled.
+*/}}
+{{- if and (not .Values.identity.enabled) .Values.identityKeycloak.enabled }}
```

Comment:
LGTM, but please add a unit test for it.

Reply:
Added the unit test but still testing as its not working yet.